package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "date-format": "^1.0.0",
     "debug": "^2.2.0",
+    "redis": "^2.6.5",
```

Comment:
don't put redis in dependencies - put it in "optionalDependencies" - not every user will need to use the redis appender.